### DISCUSSION TITLE
Using `require` to add `remark` or `rehype` plugins is unsupported, but the recommended way in documentation.

### DISCUSSION THREAD

**Discussion by ayhon** [gatsbyjs/gatsby#34098] [Help]
Trying to add some plugins to render math equations in my blog (Created with the theme lekoarts/gatsby-theme-minimal-blog), I used the recommended way in the [documentation](https://www.gatsbyjs.com/plugins/gatsby-plugin-mdx/#remark-plugins)

```js
        ...
        remarkPlugins: [
          require("remark-abbr"),
          // To pass options, use a 2-element array with the
          // configuration in an object in the second element
          [require("remark-external-links"), { target: false }],
        ],
        ...
```

However, Gatsby complains about the use of `require` in `gatsby-config.js`
```
 Error: [ERR_REQUIRE_ESM]: require() of ES Module /home/ayhon/git/gatsby-blog/node_mo
  dules/remark-math/index.js from /home/ayhon/git/gatsby-blog/gatsby-config.js not sup
  ported.
```

If `require` is not supported in the `gatsby-config.js` file, I believe it shouldn't appear as a recommended way of adding plugins in the documentation. 

Looking around, I couldn't find a way to provide the plugins to `gatsby-plugin-mdx`.

**Comment by LekoArts** [comment]
You're using a version of remark plugins that is ESM only. You'll have to downgrade remark plugins by one major version to use the CommonJS version.

**Reply by ayhon** [reply]
How do I downgrade? Should I also downgrade `gatsby`? It's in version 4

**Reply by LekoArts** [reply]
Edit your `package.json` to use the previous major version, e.g.
`"remark-math": "^5.0.0"` => `"remark-math": "^4.0.0"`

**Reply by fzyzcjy** [reply]
Hi, I wonder how can I use the latest version of `remark-math`? I have tried `require` and `import` but both give errors...
